test/e2e/scripts/run-instance/22-argo-submit.sh

```diff
@@ -61,7 +61,7 @@ case "$ARGO_WORKFLOW" in
         kubectl create secret generic dd-keys --from-literal=DD_API_KEY=123er --from-literal=DD_APP_KEY=123er1
 
         ./argo template create ../../argo-workflows/templates/*.yaml
-        ./argo submit ../../argo-workflows/${ARGO_WORKFLOW}-workflow.yaml --wait \
+        ./argo submit ../../argo-workflows/"${ARGO_WORKFLOW}"-workflow.yaml --wait \
```

Comment:
Should the whole string be quoted instead:
```
"../../argo-workflows/${ARGO_WORKFLOW}-workflow.yaml"
```

Reply:
This was done by `shellcheck` automatically, but I agree with you your suggestion looks better. See #10217